doc/md/icp-features/3-caller-identification.md

```diff
@@ -0,0 +1,389 @@
+---
+sidebar_position: 3
+---
+
+# Caller identification
+
+[Principals](https://internetcomputer.org/docs/building-apps/canister-management/control) serve as unique identifiers for users, canisters, and other entities on ICP. Every user interacting with a canister will have a principal which acts as their [identity](https://internetcomputer.org/docs/building-apps/getting-started/identities) when making requests. Principals are privacy-preserving methods of authentication, meaning they can vary across different applications, preventing a user’s identity from being easily tracked across multiple dapps.
```

Comment:
Maybe this is a better link: https://internetcomputer.org/docs/building-apps/essentials/canisters#canister-id
Because controller principals are a bit too specific. Caller can be any principal (anonymous, from agent, another canister).

Reply:
Similarly below.